Amazon.QLDB.Driver/client/Session.cs

```diff
@@ -57,15 +58,15 @@ internal class Session
         }
 
         /// <summary>
-        /// Factory method for constructing a new Session, creating a new session to QLDB on construction.
+        /// Async factory method for constructing a new Session, creating a new session to QLDB on construction.
         /// </summary>
         ///
         /// <param name="ledgerName">The name of the ledger to create a session to.</param>
         /// <param name="sessionClient">The low-level session used for communication with QLDB.</param>
         /// <param name="logger">The logger to inject any logging framework.</param>
         ///
         /// <returns>A newly created <see cref="Session"/>.</returns>
-        internal static Session StartSession(string ledgerName, AmazonQLDBSessionClient sessionClient, ILogger logger)
+        internal static async Task<Session> StartSessionAsync(string ledgerName, AmazonQLDBSessionClient sessionClient, ILogger logger)
```

Comment:
Missing the cancellation token. Check this for all the async methods

Reply:
Added token.